Program.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace AnimalSounds
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var animals = new List<Animal>
+            {
+                new Animal { Type = "Dog", Sound = "Bark" },

```

Comment:
new Animal("Batoza", "VRUM VRUM"), din nou, unde este incapsularea? de fapt in clasa Animal pot pune orice.